src/main/java/org/akaza/openclinica/controller/ScheduledExtractController.java

```diff
@@ -0,0 +1,84 @@
+package org.akaza.openclinica.controller;
+
+import core.org.akaza.openclinica.bean.login.UserAccountBean;
+import core.org.akaza.openclinica.dao.core.CoreResources;
+import core.org.akaza.openclinica.dao.extract.ArchivedDatasetFileDAO;
+import core.org.akaza.openclinica.dao.hibernate.StudyDao;
+import core.org.akaza.openclinica.domain.datamap.Study;
+import core.org.akaza.openclinica.service.UtilService;
+import core.org.akaza.openclinica.service.extract.XsltTriggerService;
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import org.quartz.*;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpRequest;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.ResponseBody;
+
+import javax.servlet.http.HttpServletRequest;
+
+@Controller
+@RequestMapping(value = "/auth/api")
+@Api(value = "ScheduledExtractJob", tags = {"Scheduled Extract Job"}, description = "REST API for Scheduled Extract Job")
+public class ScheduledExtractController {
+
+    private final static Logger logger = LoggerFactory.getLogger(ScheduledJobController.class);
+
+    @Autowired
+    private UtilService utilService;
+
+    @Autowired
+    private Scheduler scheduler;
+
+    @Autowired
+    private StudyDao studyDao;
+
+
+
+
+    @ApiOperation(value = "To get scheduled extract jobs for specific job name at study level", notes = "only work for authorized users with the right access permission ")
+    @RequestMapping(value = "/studies/{studyOID}/extractJobs/{jobName}/result", method = RequestMethod.GET)
```

Comment:
This API is going to change given the recent updates, right?

Reply:
Yeah ignore this pr